content/terraform/v1.10.x/docs/intro/use-cases.mdx

```diff
@@ -48,7 +48,7 @@ Terraform can help you enforce policies on the types of resources teams can prov
 
 ### Resources
 
-- Try the [Control Costs with Policies](/terraform/tutorials/cloud-get-started/cost-estimation) tutorial to estimate the cost of infrastructure changes and define policy to limit it.
+- The [cost estimation documentation](/terraform/cloud-docs/cost-estimation) provides information to estimate the cost of infrastructure changes and define policies to limit it.
```

Comment:
```suggestion
- The [cost estimation documentation](/terraform/cloud-docs/cost-estimation) describes how to use the cost estimation feature and how to define policies that limit costs associated with infrastructure changes.
```
It might be silly, but the original wording sounds as if the docs actually has cost estimation and policy data.